entries/slotfills/show-generator-button/sidebar.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
```

Comment:
Use the React version bundled in Gutenberg.

```suggestion
import { useState } from '@wordpress/element';
```